compiler/modules/types/float.bal

```diff
@@ -0,0 +1,61 @@
+// Implementation specific to basic type float.
+
+type FloatSubtype readonly & record {|
+    boolean allowed;
+    float[] values;
+|};
+
+public function floatConst(float value) returns SemType {
+    FloatSubtype st = { allowed: true, values: [value] };
+    return uniformSubtype(UT_FLOAT, st);
+}
+
+function floatSubtypeUnion(SubtypeData d1, SubtypeData d2) returns SubtypeData =>
```

Comment:
Don't see why we need this. Can't we just put enumerableSubtypeUnion in the vtable?